### ISSUE TITLE
Jetpack Search: revise the Thank you note upon purchase

### ISSUE THREAD

**Issue by AnnaMag** [Automattic/wp-calypso#40434]
#### Steps to reproduce
Follow the steps from https://github.com/Automattic/wp-calypso/pull/40360

Revise the copy and verify the redirect.

**Comment by keoshi** [comment]
Proposed copy for the thank you modal, which reuses copy from https://github.com/Automattic/jetpack/pull/15074

> **Welcome to Jetpack Search**
>
> We are currently indexing your site and will notify you when it is complete.
>
> In the meantime, we have added some common filtering widgets to your site that you should try customizing in your traditional WordPress dashboard.
> 
> [Customize Search now (opens the respective wp-admin link)] [I’ll do it later (closes modal)]

How does this feel?

**Comment by gibrown** [comment]
@keoshi We don't have "We are currently indexing your site and will notify you when it is complete." implemented anywhere. I think we should just say that we are indexing it.

**Comment by keoshi** [comment]
@gibrown As an MVP we could send the customer an email once the indexing is complete. If that's out of scope for now, then let's remove that “will notify you” mention on both of these.

**Comment by gibrown** [comment]
Ya it is out of scope for the moment.
